### PR TITLE
Source Fastbill: Migrate to manifest-only format

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
Source Fastbill: Migrate to manifest-only formats with components

## How
Migrated to inline schemas
Migrated spec
Migrated to manifest only with ci

### CODE REVIEW

File: airbyte-integrations/connectors/source-fastbill/acceptance-test-config.yml [hunk 1]
```diff
@@ -3,9 +3,9 @@
...
 acceptance_tests:
-  # spec:
-  #   tests:
-  #     - spec_path: "source_fastbill/spec.yaml"
+  spec:
+    tests:
+      - spec_path: "manifest.yaml"
   connection:
     tests:
       #     - config_path: "secrets/config.json"
```
Comment (ChristoGrab) [inline airbyte-integrations/connectors/source-fastbill/acceptance-test-config.yml:11]: I created a new sandbox trial account for the connector. Let's go ahead and uncomment these tests

File: airbyte-integrations/connectors/source-fastbill/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,2770 @@
...
+    type: DeclarativeStream
+    retriever:
+      type: SimpleRetriever
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path:
+          - "RESPONSE"
+          - "{{ parameters.record_extractor }}"
```
Comment (ChristoGrab) [inline airbyte-integrations/connectors/source-fastbill/manifest.yaml:805]: This is indicative of a bug in the migration script 😕 The use of parameters isn't compatible with the builder UI, so we have to resolve them when migrating to manifest-only. This should happen automatically but I'm seeing more examples where it failed. I'm looking into the issue, but for now we can try this flow to resolve:

1. Grab this manifest, throw it in the builder and switch to UI mode.
2. Look for any fields that reference parameters. Check the existing manifest in Github and update the values for each stream ([example here](This is a bug in the migration script 😕 The use of parameters isn't compatible with the builder UI, so we have to resolve them when migrating to manifest-only. This should happen automatically but I'm seeing more examples where it failed. I'm looking into the issue, but for now we can try this flow to resolve:

1. Grab this manifest, throw it in the builder and switch to UI mode.
2. Look for any fields that reference parameters. Check the existing manifest in Github and update the values for each stream [(example here)](https://github.com/airbytehq/airbyte/blob/1c2f72ec09200b0a3bc04f2fd37b990e9dadd82d/airbyte-integrations/connectors/source-fastbill/source_fastbill/manifest.yaml#L48)
3. Switch back to YAML view and copy the updated manifest for the PR.

Hopefully that will resolve it. Let me know if anything about this is unclear!

**Comment by ChristoGrab** [comment]
@btkcodedev It looks like the metadata didn't properly update during the migration, as the baseImage is still set to python-connector-base; that's what's causing the build failure. Let's update it to source-declarative-manifest, you can use:

baseImage: docker.io/airbyte/source-declarative-manifest:5.15.0@sha256:09a84e0622f36393077332faf11cc239e77083fae5fa500592c049dca25888a7

Then we'll want to uncomment the acceptance tests and resolve the parameters in the manifest as described in the review comments 👍

File: airbyte-integrations/connectors/source-fastbill/manifest.yaml (hunk 2 above)
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-fastbill/manifest.yaml]: Updates done!, Hopefully it works

File: airbyte-integrations/connectors/source-fastbill/acceptance-test-config.yml (hunk 1 above)
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-fastbill/acceptance-test-config.yml:11]: Cool!

File: airbyte-integrations/connectors/source-fastbill/manifest.yaml (hunk 2 above)
Comment (ChristoGrab) [reply airbyte-integrations/connectors/source-fastbill/manifest.yaml]: Looks like that fixed it, nice work @btkcodedev!

**Review by ChristoGrab — APPROVED** [review]

**Pull request merged**